src/EditorFeatures/Core/Shared/Utilities/CommonFormattingHelpers.cs

```diff
@@ -12,17 +12,7 @@ namespace Microsoft.CodeAnalysis.Editor.Shared.Utilities
     internal static class CommonFormattingHelpers
     {
         public static TextSpan GetFormattingSpan(SyntaxNode root, TextSpan span)
-        {
```

Comment:
Moved down.  I kept around a forwarding method because some code still refers to helpers in here, and since the type name is the same, they couldn't be updated to point at the new location without causing an ambiguity problem.